torch/__init__.py

```diff
@@ -157,15 +157,17 @@ def set_default_dtype(d):
     used as default floating point type for type inference in
     :func:`torch.tensor`.
 
+    The default floating point dtype is initially ``torch.float32``.
+
     Args:
         d (:class:`torch.dtype`): the floating point dtype to make the default
 
     Example::
 
-        >>> torch.tensor([1.2, 3]).dtype  # default is float32
+        >>> torch.tensor([1.2, 3]).dtype           # initial default for floating point is torch.float32
         torch.float32
-        >>> torch.set_default_dtype(torch.double)
-        >>> torch.tensor([1.2, 3]).dtype
+        >>> torch.set_default_dtype(torch.double)  # set to torch.float64
```

Comment:
why use two aliases for the same thing and then explain the alias instead of just using the main name?